### DISCUSSION TITLE
《回归奖励》

### DISCUSSION THREAD

**Discussion by YuTinMIn** [share121/inter-knot#2542] [其他]
![双金](https://github.com/user-attachments/assets/6f56390d-22ac-489e-97d5-0624bd785f78)
我ji已经几个月没有玩原神了，都在玩绝区零和崩铁，然后我听说原神要送五星了，于是我云玩看一眼，发现还没有开始活动，然后我就用邮箱里的十连抽了一发，结果就双金了，可能是老米给的回归奖励吧，这个老米人真好啊（咸鱼音）
![Uploading 咸鱼.png…]()

**Comment by hua-yuanshuai** [comment]
为什么要发两个一模一样的帖子<img src="https://bbs-static.miyoushe.com/static/2024/01/31/a065264ccf369d43f39162320393d9bf_5251148779871794599.png" alt="花火 我来啦" width="75" >

**Reply by YuTinMIn** [reply]
因为这个帖子有一张图片没有加载出来，但是不知道为什么删不掉，所以就有了两个帖子，这一个帖子是要删掉的

**Comment by Brighttaz** [comment]
不知道为什么，跟回坑有保护期似的，退一会回来就能出金（）
